app/javascript/components/widgets/widgets/land-cover/tree-cover/selectors.js

```diff
@@ -16,11 +16,12 @@ export const parseData = createSelector(
   [getData, getSettings, getWhitelist, getColors, getCurrentLocation],
   (data, settings, whitelist, colors, currentLabel) => {
     if (isEmpty(data)) return null;
-    const { totalArea, cover, plantations } = data;
+    const { totalArea, cover } = data;
     const { indicator } = settings;
     const hasPlantations =
       (currentLabel !== 'global' && isEmpty(whitelist)) ||
       whitelist.indexOf('plantations') > -1;
+    const plantations = hasPlantations ? data.plantations : 0;
```

Comment:
why not get plantations from data still? If you are getting totalArea etc might as well do it the same way. Just being picky.

Reply:
See update.